docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -6,43 +6,43 @@
 secure, cross‐platform and self-contained.
 Snaps can be installed on any Linux distribution with [snap support](https://snapcraft.io/docs/installing-snapd).
 
-Snap packages of EdgeX services are published on the [Snap Store](https://snapcraft.io).
-
-The main platform snap is called the [edgexfoundry](https://snapcraft.io/edgexfoundry). It contains
-all reference core services along with several other security, supporting, application, and device services.
-
-The list of all EdgeX snaps is available [below](#edgex-snaps).
+Snap packages of EdgeX services are published on the [Snap Store](https://snapcraft.io). The list of all EdgeX snaps is available [below](#edgex-snaps).
 
 ## EdgeX Snaps
 The following snaps are maintained by the EdgeX working groups:
 
 Platform snap:
 
-- [EdgeX Foundry](https://snapcraft.io/edgexfoundry)
+- [edgexfoundry](https://snapcraft.io/edgexfoundry): the main platform snap containing
+all reference core services along with several other security, supporting, application, and device services.
 
 Development tools:
 
-- [EdgeX UI](https://snapcraft.io/edgex-ui)
-- [EdgeX CLI](https://snapcraft.io/edgex-cli)
+- [edgex-ui](https://snapcraft.io/edgex-ui)
+- [edgex-cli](https://snapcraft.io/edgex-cli)
 
 Application services:
 
-- [App Service Configurable](https://snapcraft.io/edgex-app-service-configurable)
+- [edgex-app-service-configurable](https://snapcraft.io/edgex-app-service-configurable)
 
 Device services:
 
-- [Device Camera](https://snapcraft.io/edgex-device-camera)
-- [Device Modbus](https://snapcraft.io/edgex-device-modbus)
-- [Device MQTT](https://snapcraft.io/edgex-device-mqtt)
-- [Device REST](https://snapcraft.io/edgex-device-rest)
-- [Device SNMP](https://snapcraft.io/edgex-device-snmp)
-- [Device Grove](https://snapcraft.io/edgex-device-grove)
+- [edgex-device-camera](https://snapcraft.io/edgex-device-camera)
+- [edgex-device-modbus](https://snapcraft.io/edgex-device-modbus)
+- [edgex-device-mqtt](https://snapcraft.io/edgex-device-mqtt)
+- [edgex-device-rest](https://snapcraft.io/edgex-device-rest)
+- [edgex-device-snmp](https://snapcraft.io/edgex-device-snmp)
+- [edgex-device-grove](https://snapcraft.io/edgex-device-grove)
 
 Other EdgeX snaps do exist on the public Snap Store ([search by keyword](https://snapcraft.io/search?q=edgex)) or private stores under brand accounts.
 
 ## Installing the `edgexfoundry` snap
+[![](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-white.svg){: style="height:4em"}](https://snapcraft.io/edgexfoundry)
+
+This is the main platform snap which
+contains all reference core services along with several other security, supporting, application, and device services.
```

Comment:
Are they duplicated with lines 16-17?

Reply:
Yes but in different sections. I kept in both places on purpose to make the installation section self-contained, in case we wanna refer to it from outside.

Reply:
Ok! I see.